frontend/tsconfig.json

```diff
@@ -8,6 +8,8 @@
     "outDir": "build",
     "resolveJsonModule": true,
     "skipLibCheck": true,
+    "module": "es2020",
```

Comment:
This is needed to use `import.meta`, which is what's used in the [webpack docs](https://webpack.js.org/guides/web-workers/). Without this you get an error of `TS1343: The 'import.meta' meta-property is only allowed when the '--module' option is 'es2020', 'es2022', 'esnext', 'system', 'node12', or 'nodenext'.`

Reply:
sounds good to me!